pkg/core/state/nep5.go

```diff
@@ -85,12 +86,18 @@ func (bs *NEP5Balances) EncodeBinary(w *io.BinWriter) {
 // Append appends single transfer to a log.
 func (lg *NEP5TransferLog) Append(tr *NEP5Transfer) error {
 	w := io.NewBufBinWriter()
+	// The first entry, set up counter.
+	if len(lg.Raw) == 0 {
```

Comment:
Looks a bit ugly, but probably can't be avoided (without making additional slice copy in `DAO.PutNEP5TransferLog`. The only other option I can think of is restricting transfer log by size in bytes (vs. by amount of transfers), but this can make RPC processing a bit complicated.